src/styles/shiki.css

```
@@ -34,28 +39,56 @@ code .line::after {
     counter-increment: line;
     content: counter(line);
 
-    @apply mr-5 inline-block w-4 text-right text-sm text-[#999999];
+    margin-right: 1.25rem;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)